src/components/Tabs/Tabs.tsx

```diff
@@ -15,6 +15,9 @@ import Tab from './Tab';
 
 export interface TabsProps {
   children: ReactElement[] | ReactElement;
+  /**
+   * Label to be read by screen readers
```

Comment:
suggest aligning with the w3c pattern descriptions where we can. in this case "label that describes the purpose of the set of tabs"